.changeset/lovely-starfishes-poke.md

```diff
@@ -0,0 +1,5 @@
+---
+"@ciena-org/ember-codemods": minor
```

Comment:
Sorry I missed this but this should be a `patch` since the package is still in the alpha release, so bugs and features are `patch` and breaking changes will be `minor`. This is specified [here](https://github.com/ciena-blueplanet/ember-codemods/blob/main/CONTRIBUTING.md?plain=1#L75)
```suggestion
"@ciena-org/ember-codemods": patch
```

Reply:
Done